mex/extractors/pipeline/base.py

```diff
@@ -28,6 +37,95 @@ def run_job_in_process(group_name: str = "default") -> "ExecutionResult":
     return job.execute_in_process()
 
 
+def create_monitor_jobs_sensor(extractor_group_names: list[str]) -> SensorDefinition:
+    """Wrapper to be able to pass group_names variable."""
+
+    @sensor(
+        job_name="publisher",
+        default_status=DefaultSensorStatus.RUNNING,
+        minimum_interval_seconds=3600,  # 60 min
+    )
+    def monitor_jobs_sensor(
+        context: SensorEvaluationContext,
+    ) -> RunRequest | SkipReason:
+        """Sensor to monitor the completion of all extractors and trigger publisher."""
+        instance = DagsterInstance.get()
+        last_run_time_publisher = context.cursor or "1970-01-01T00:00:00+00:00"
+        latest_start_time_extractors = last_run_time_publisher
+        completed = True
+
+        jobs_are_running = instance.get_runs(
+            filters=RunsFilter(
+                statuses=[
+                    DagsterRunStatus.STARTING,
+                    DagsterRunStatus.STARTED,
+                    DagsterRunStatus.CANCELING,
+                    DagsterRunStatus.QUEUED,
+                    DagsterRunStatus.NOT_STARTED,
+                ],
+            )
+        )
+        if jobs_are_running:
+            return SkipReason(
+                "No publishing because other jobs are running at the moment."
+            )
+
+        for group in extractor_group_names:
+            runs = instance.get_runs(
+                filters=RunsFilter(
+                    job_name=group,
+                    statuses=[
+                        DagsterRunStatus.SUCCESS,
+                        DagsterRunStatus.FAILURE,
+                        DagsterRunStatus.CANCELED,
+                    ],
+                )
+            )
+
+            recent_run = max(
+                (
+                    run
+                    for run in runs
+                    if ".dagster/scheduled_execution_time" in run.tags
+                    and run.tags[".dagster/scheduled_execution_time"]
+                    > last_run_time_publisher
+                ),
+                key=lambda run: run.tags[".dagster/scheduled_execution_time"],
+                default=None,
+            )
+
+            # Update completed flag based on whether a recent run was found
+            if recent_run is None or recent_run.status not in [
+                DagsterRunStatus.SUCCESS,
+                DagsterRunStatus.FAILURE,
+                DagsterRunStatus.CANCELED,
+            ]:
```

Comment:
You already check for the status while getting the runs.

```suggestion
            if recent_run is None:
```